gitlab/datadog_checks/gitlab/gitlab.py

```diff
@@ -80,7 +82,7 @@ def _create_gitlab_prometheus_instance(self, instance, init_config):
         gitlab_instance.update(
             {
                 'namespace': 'gitlab',
-                'metrics': allowed_metrics,
+                'metrics': metrics,
                 # Defaults that were set when gitlab was based on PrometheusCheck
                 'send_monotonic_counter': instance.get('send_monotonic_counter', False),
                 'health_service_check': instance.get('health_service_check', False),
```

Comment:
Let's also enable `send_distribution_counts_as_monotonic` as this is the correct behavior (submits summary counter metrics as `monotonic_count`).

Reply:
Ok!